azure-iot-device/tests/common/pipeline_test_fixtures.py

```diff
@@ -0,0 +1,25 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
```

Comment:
Again, I'm not really sure why this is in tests.common rather than tests.common.pipeline, given that it's specific for pipeline tests.

Is it because you expose it through conftest.py? If so, did you know that you can have a conftest in every directory/subdirectory that applies to everything from that level down?

i.e. it seems like this pipeline stuff in tests.common should all be in tests.common.pipeline, including a new conftest.py

Reply:
Agree that it's awkward.  Will revisit in a later PR.

---
In reply to: [287541060](https://github.com/Azure/azure-iot-sdk-python-preview/pull/94#discussion_r287541060) [](ancestors = 287541060)